jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -808,13 +809,15 @@ code_blockt get_user_specified_clinit_body(
             }
           }
         }
+        if(symbol_table.lookup(real_clinit_name) == nullptr)
+          return code_blockt{};
```

Comment:
Add a comment here about how/why this could happen

Reply:
done